processor/maskprocessor/processor.go

```diff
@@ -211,3 +211,27 @@ func (p *maskProcessor) createMaskFunc(field string) func(k string, v pcommon.Va
 		return true
 	}
 }
+
+// createRules creates a map of rules for the processor.
+func (p *maskProcessor) createRules() (map[string]*regexp.Regexp, error) {
+	if len(p.cfg.Rules) == 0 {
+		return compileRules(defaultRules)
```

Comment:
Can we store the default rules as a 'map[string]*regexp`? Using must compile to seed it? They're default so we shouldn't take any chances of them but working.